### PR TITLE
CI: add Zephyr build test for RISCV architecture

### PR DESCRIPTION
**Pull request by arnopo**
Add CI build test for the RISCV32 and RISCV64 architecture.

Signed-off-by: Arnaud Pouliquen <arnaud.pouliquen@foss.st.com>

### CODE REVIEW

**Comment by carlocaione** [comment]
Uhm, this is failing. I'll take a look.

File: .github/actions/build_ci/entrypoint.sh [hunk 1]
```diff
@@ -74,8 +74,17 @@ build_zephyr(){
     	cd ./zephyr &&
     	source zephyr-env.sh &&
 	cd ../.. &&
-	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr &&
-	cd build-zephyr &&
+	echo  "###### Build for qemu_cortex_m3 ######" &&
+	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr-m3 &&
+	cd build-zephyr-m3 &&
+	make VERBOSE=1 &&
+	echo  "###### Build for qemu_riscv64 ######" &&
```
Comment (carlocaione) [inline .github/actions/build_ci/entrypoint.sh:81]: probably unrelated to the failure but we need to get out from `build-zephyr-m3` before starting the building for `riscv` targets.

Comment (carlocaione) [reply .github/actions/build_ci/entrypoint.sh:81]: uhm, probably not unrelated because it's working for me locally.

Comment (arnopo) [reply .github/actions/build_ci/entrypoint.sh:81]: Thanks for pointing out the issue!
What i can figure out is why it did not failed when i tested on my github (https://github.com/arnopo/libmetal/runs/4700942696?check_suite_focus=true)
i will fix it

File: .github/actions/build_ci/entrypoint.sh [hunk 2]
```diff
@@ -74,8 +74,20 @@ build_zephyr(){
...
-	cd build-zephyr &&
+	echo  "###### Build for qemu_cortex_m3 ######" &&
+	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_cortex_m3 -DWITH_TESTS=on -Bbuild-zephyr-m3 &&
+	cd build-zephyr-m3 &&
+	make VERBOSE=1 &&
+	cd .. &&
+	echo  "###### Build for qemu_riscv64 ######" &&
+	cmake . -DWITH_ZEPHYR=on -DBOARD=qemu_riscv64 -DWITH_TESTS=on -Bbuild-zephyr-rscv64 &&
+	cd .. &&
+	cd build-zephyr-rscv64 &&
```
Comment (carlocaione) [inline .github/actions/build_ci/entrypoint.sh:85]:
```suggestion
	cd build-zephyr-rscv64 &&
```

File: .github/actions/build_ci/entrypoint.sh (hunk 1 above)
Comment (carlocaione) [reply .github/actions/build_ci/entrypoint.sh:82]: Still something to fix.

**Review by edmooring — APPROVED** [review]
Looks good to go.

**Review by carlocaione — APPROVED** [review]

**Pull request merged**